src/ocm/services/apis/ClustersAPI.ts

```diff
@@ -118,6 +119,9 @@ const ClustersAPI = {
   },
 
   register(params: ClusterCreateParams) {
+    if (params.cpuArchitecture === CpuArchitecture.ARM) {
+      params.userManagedNetworking = true;
+    }
```

Comment:
This logic doesn't belong here, the ClustersAPI is meant to be a simple REST client.
Please perform this logic [inside the caller](https://github.com/openshift-assisted/assisted-ui-lib/blob/e1d13b7d3e82dd933d82090afec479e957e151b3/src/ocm/services/ClusterDetailsService.ts#L9) of this function.